apps/backend/src/mutations/SEPMutations.ts

```diff
@@ -274,7 +272,6 @@ export default class SEPMutations {
     return true;
   }
 
-  @Authorized([Roles.USER_OFFICER])

```

Comment:
I am concerned a bit about this removal of the authorization check. Can't we keep that somehow because I know that it can be used from outside and like this anyone can assign proposal to a SEP I guess. Correct me if I am wrong.

Reply:
@yoganandaness I am thinking if the one can send something like this:

`

await sepMutation.assignProposalsToSEPUsingCallInstrument(
        {
          id: 0,
          currentRole: {
            id: 1,
            shortCode: Roles.USER_OFFICER,
            title: 'UserOfficer',
          },
        } as UserWithRole,
        {
          instrumentId: instrumentId,
          proposalPks: [proposal.primaryKey],
        }
      );

`

Instead of null for the `agent` we send some data just to make `Authorized` decorator happy internally.